flow/scripts/global_place.tcl

```diff
@@ -48,16 +48,19 @@ if {[info exist ::env(PLACE_DENSITY_LB_ADDON)]} {
   set place_density $::env(PLACE_DENSITY)
 }
 
-if { 0 != [llength [array get ::env GLOBAL_PLACEMENT_ARGS]] } {
```

Comment:
GLOBAL_PLACEMENT_ARGS seems to be lost

Reply:
this was intentional as GLOBAL_PLACEMENT_ARGS was only used to turn on -timing driven. However I will bring this back as it can be used to turn on the other options of global_placement that are not being used.

Reply:
See https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/7aafdfab709228d553f37807aba84bcc8981daf9/flow/designs/gf12/gcd/config.mk#L12

I think that should be removed though so if you can make that change and it works then I would be glad to be rid of this var.